src/_playground/documentation/Properties/_PropertyTable.js

```diff
@@ -29,21 +29,22 @@ const PropertyTable = ({ title, propTypes, defaultProps, propDescriptions }) =>
     };
 
     sortedProps.forEach(({ propName }) => {
-        data.push({
-            rowData: [
-                propName,
-                <PropertyType componentName={title} prop={propTypes[propName]} />,
-                <PropertyDefault
-                    defaultValue={defaultProps && defaultProps[propName]}
-                    prop={propTypes[propName]} />,
-                <PropertyDescription
-                    defaultValue={defaultProps && defaultProps[propName]}
-                    description={mergedPropDescriptions[propName]}
-                    prop={propTypes[propName]} />
-            ]
-        });
         if (propTypes[propName].typeName === 'i18n') {
             shapes.push(propName);
+        } else {
+            data.push({
+                rowData: [
+                    propName,
+                    <PropertyType componentName={title} prop={propTypes[propName]} />,
+                    <PropertyDefault
+                        defaultValue={defaultProps && defaultProps[propName]}
+                        prop={propTypes[propName]} />,
+                    <PropertyDescription
+                        defaultValue={defaultProps && defaultProps[propName]}
+                        description={mergedPropDescriptions[propName]}
+                        prop={propTypes[propName]} />
+                ]
+            });
```

Comment:
I see where you're going with this and on the surface, it's not a bad idea.  The problem, however, is there is no entry in the main table of props called `localizedText` anymore.  And there _should_ be since that will be necessary for consumers to properly construct their props.  You and I know it's called `localizedText`, but not all consumers will.  Also, what's to stop us from changing that top-level prop name since all it's really tied to is the `i18n` custom prop type validator.

Additionally, I think this same type of pattern will be extended to `shape` prop types.